class7-week3/index.js

```diff
@@ -5,11 +5,15 @@ const app = Express()
 
 app.use(bodyParser.json())
 
-const {list, create, update, remove} = require('./actions')
+const {list, create, update, remove, reset, markAsDone, markAsNotDone} = require('./actions')
 
 app.get('/todos', list)
 app.post('/todos', create)
+app.put('/todos/:id/done',markAsDone)
```

Comment:
I agree with your thinking - this sounds like a PUT/UPDATE to me too. However, the homework asked for this to be a POST. :-)